config/clients/js/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v0.3.2
+
+### [0.3.1](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.3.1...v0.3.2) (2024-02-13)
+
+- feat: add example project
+- feat: add support for `apiUrl` configuration option
```

Comment:
Can you add that `apiScheme` and `apiHost` are considered deprecated now?

Reply:
Edited and amended the js-sdk commit with this change